client.go

```diff
@@ -77,6 +89,40 @@ func NewClient(ctx *Context, cfg *Config, w *Wallet) (*Client, error) {
 	return &Client{cfg: cfg, ethClient: ethClient, client: c, w: w.w, onChain: acc, dialer: dialer}, nil
 }
 
+// Close closes the client and its PersistRestorer to synchronize the database.
+// ref https://pkg.go.dev/perun.network/go-perun/client?tab=doc#Channel.Close
+// ref https://pkg.go.dev/perun.network/go-perun/channel/persistence/keyvalue?tab=doc#PersistRestorer.Close
+func (c *Client) Close() {
+	c.client.Close()
+}
+
+// Handle is the handler routine for channel proposals and channel updates. It
+// must only be started at most once by the user.
+// Incoming proposals and updates are forwarded to the passed handlers.
+// ref https://pkg.go.dev/perun.network/go-perun/client?tab=doc#Client.Handle
+func (c *Client) Handle(ph ProposalHandler, uh UpdateHandler) {
+	c.client.Handle(&proposalHandler{c: c, h: ph}, &updateHandler{h: uh})
+}
+
+// OnNewChannel sets a handler to be called whenever a new channel is created
+// or restored. Only one such handler can be set at a time, and repeated calls
+// to this function will overwrite the currently existing handler. This
+// function may be safely called at any time.
+// Start the watcher routine here, if needed.
+// ref https://pkg.go.dev/perun.network/go-perun/client?tab=doc#Client.OnNewChannel
+func (c *Client) OnNewChannel(acc ChannelAcceptor) {
+	c.mtx.Lock()
+	defer c.mtx.Unlock()
+
+	c.acceptor = acc
```

Comment:
Remove `c.acceptor` field from type `Client`. Always keep as few state as possible. It is only used here locally to set the `OnNewChannel` callback.